DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/convolution2d/src/convolution_kernel.hpp

```diff
@@ -21,6 +21,12 @@ constexpr int kKernelVersion = 1;
 // Define input/output streaming interfaces
 /////////////////////////////////////////////
 
+#if defined(FPGA_SIMULATOR) || defined(FPGA_HARDWARE)
+constexpr size_t kPipeMinCapacity = 0;
+#else  // #if defined(FPGA_EMULATOR)
+constexpr size_t kPipeMinCapacity = 15000;
+#endif
```

Comment:
this extra boilerplate is unintuitive and must be explained in the Readme.

Reply:
streaming_data_interfaces/README.md has been updated to explain the capacity. The explanation is common and also apply to this sample.